charts/nginx-gateway-fabric/templates/certs-job.yaml

```diff
@@ -153,6 +153,7 @@ spec:
       securityContext:
         fsGroup: 1001
         runAsNonRoot: true
+  ttlSecondsAfterFinished: {{ .Values.certGenerator.ttlSecondsAfterFinished }}
```

Comment:
This should be at the end where it was, so it's not in the middle of the container spec.